src/App.js

```diff
@@ -19,7 +19,8 @@ function App() {
         <Route path="/restaurants" element={<Restaurants />} />
         <Route path="/register-restaurant" element={<RegisterRestaurants />} />
         <Route path="/orders" element={<Orders />} />
-        <Route path="/restaurant-details" element ={ <RestaurantDetails/>} />
+        <Route path="/restaurant-details" element={<RestaurantDetails />} />
+        <Route path="/menu-details" element={<Menudetails />} />
```

Comment:
Just curious. Why are we having two different routes for restaurant details and menu details?

Reply:
So that we don't have any merge conflict at the end

Reply:
Made the changes.